Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8.6-slim-buster AS base
+RUN apt-get update -y && apt-get install -y curl 
+# COPY REQUIREMENTS
+WORKDIR /app
+
+# COPY .env .env - configuration secret
+COPY poetry.toml poetry.toml
+COPY pyproject.toml pyproject.toml
+COPY poetry.lock poetry.lock
+
+# virtual env
+ENV POETRY_HOME=/poetry
+ENV PATH=${POETRY_HOME}/bin:${PATH}
+# declar env poetry -- 
+# INSTALL DEPENDENCIES
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+RUN poetry config virtualenvs.create false --local
+RUN poetry install
+
+# PROD IMAGE
+    FROM base as production
+    RUN pip install gunicorn
+    EXPOSE 5000
+#    COPY todo_app todo_app - no more copying the code.
```

Comment:
I think this is commented out because you're mounting a volume so the code gets copied across anyway - that works here, but usually we would explicitly copy the code at least for production, as otherwise our docker container can only be run by somebody who already has the code to hand, rather than being a reasonably stand-alone artifact. I think we probably want to add this back in

Reply:
I have added copy step only for production image, assuming production image may run on remote server, etc.